lib/modules.nix

```diff
@@ -404,7 +404,10 @@ let
           };
         in parentFile: parentKey: initialModules: args: collectResults (imap1 (n: x:
           let
-            module = checkModule (loadModule args parentFile "${parentKey}:anon-${toString n}" x);
+            module =
+              builtins.addErrorContext
+                "while loading imports of module `${parentFile}`"
```

Comment:
Nix would compulsively start with `while evaluating`, but does very little except reduce the signal to noise ratio. We can use a different verb that actually conveys information.